### DISCUSSION TITLE
support for pain.001 version 14?

### DISCUSSION THREAD

**Discussion by acho-bacho** [prowide/prowide-iso20022#16] [Q&A] [answered]
Hello and thanks for the wonderful project!

Do you plan to add support for version 14 of pain.001 format?

Thanks!

**Comment by zubri** [comment]
Yes, we have noticed there are some missing versions in the SRU2022 release.

**Comment by zubri** [comment, marked as answer]
We have added many missing types in release SRU2022-9.3.4

However, there is no pain.001.001.14, the latest version available in the current published standard is 11.
